packages/components/src/components/as-histogram-widget/as-histogram-widget.e2e.ts

```diff
@@ -132,9 +173,30 @@ const histogramDataFloatingToDecimal = [
   { start: 3.8901, end: 3.9081, value: 20 },
 ];
 
-const histogramDataFloatingToSIPrefix = [
-  { start: 0.1234, end: 0.1432, value: 5 },
-  { start: 0.2345, end: 0.4543, value: 10 },
-  { start: 0.5678, end: 0.7876, value: 15 },
-  { start: 0.8901, end: 0.9081, value: 20 },
+const histogramDataFloatingToDecimalPrecision2 = [
+  { start: 0.12, end: 0.14, value: 5 },
+  { start: 0.23, end: 0.45, value: 10 },
+  { start: 0.56, end: 0.78, value: 15 },
+  { start: 0.89, end: 0.90, value: 20 },
+];
+
+const histogramDataFloatingToDecimalPrecision3 = [
+  { start: 0.124, end: 0.125, value: 5 },
+  { start: 0.125, end: 0.126, value: 10 },
+  { start: 0.126, end: 0.127, value: 15 },
+  { start: 0.127, end: 0.128, value: 20 },
+];
+
+const histogramDataFloatingToDecimalPrecision6 = [
+  { start: 0.12341234, end: 0.14321432, value: 5 },
+  { start: 0.23452345, end: 0.45434543, value: 10 },
+  { start: 0.56785678, end: 0.78767876, value: 15 },
+  { start: 0.89018901, end: 0.90819081, value: 20 },
+];
+
+const histogramDataFloatingToDecimalMicro = [
```

Comment:
A picky thing: if this data constants are just used in one spec, IMO they should be inside of it, near the asserts. That way it would be easier to quickly understand the spec without scrolling.